backend/src/main/java/aiversum/backend/rest/UserController.java

```diff
@@ -0,0 +1,34 @@
+package aiversum.backend.rest;
+
+import aiversum.backend.model.User;
+import aiversum.backend.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping(path = "/user")
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/register")
+    public ResponseEntity<?> createUser(@RequestBody User user) {
+
+        Optional<User> existingUser = userService.findByEmail(user.getEmail());
```

Comment:
I think you should do that in the service and throw an exception over there. There you can catch the exception here and return CONFLICT